src/AasxPackageExplorer.GuiTests/Common.cs

```diff
@@ -271,6 +292,57 @@ public static void AssertLoadAasx(Application application, Window mainWindow, st
                     $"Check manually why the file could not be opened: {path}");
         }
 
+        public static Window FindTopLevelWindow(Application application, UIA3Automation automation,
+                Func<Window, bool> filter, string timeoutMessage, int timeoutSeconds = 5)
+        {
+            return Retry.WhileNull(() =>
+                    // ReSharper disable once AccessToDisposedClosure
+                    application.GetAllTopLevelWindows(automation).FirstOrDefault(filter),
+                throwOnTimeout: true, timeout: TimeSpan.FromSeconds(timeoutSeconds), timeoutMessage: timeoutMessage
+            ).Result;
+        }
+
+        public static Window FindTopLevelWindowByTitle(Application application, UIA3Automation automation,
+                string title, int timeoutSeconds = 5)
+        {
+            return FindTopLevelWindow(application, automation, (w) => w.Title == title,
+                    $"Could not find the top-level window with the title '{title}'", timeoutSeconds);
```

Comment:
A small nitpick: use `Quote` here for the title so that the special characters are properly escaped:

```suggestion
                    $"Could not find the top-level window with the title {Quote(title)}", timeoutSeconds);
```